test/row.jl

```diff
@@ -101,21 +101,12 @@ end
 end
 
 @testset "`ClassRow" begin
-    @test isa(Lighthouse.ClassRow(; class_index=3).class_index, Int64)
-    @test isa(Lighthouse.ClassRow(; class_index=Int8(3)).class_index, Int64)
-    @test Lighthouse.ClassRow(; class_index=:multiclass).class_index == :multiclass
+    @test isa(Lighthouse.ClassRow(; class_index=3, class_labels=missing).class_index, Int64)
+    @test isa(Lighthouse.ClassRow(; class_index=Int8(3), class_labels=missing).class_index, Int64)
+    @test Lighthouse.ClassRow(; class_index=:multiclass, class_labels=["a, "b]).class_labels == ["a", "b"]
```

Comment:
Oops, my previous suggestion was bad; I didn't mean to change the old test but simply add a new one:
```suggestion
    @test Lighthouse.ClassRow(; class_index=:multiclass).class_index == :multiclass
    @test Lighthouse.ClassRow(; class_index=:multiclass, class_labels=["a, "b]).class_labels == ["a", "b"]
```

Reply:
Good catch!!